msteams-platform/toolkit/configure-message-extension-capability.md

```diff
@@ -0,0 +1,634 @@
+       "value": "${{SECRET_BOT_PASSWORD}}"
+     },
+     "botDisplayName": {
+       "value": "bot"
+     },
+     "resourceBaseName2":{
+       "value": "bot${{RESOURCE_SUFFIX}}"
+     }
+   }
+   ```
+
+1. Add `botAadApp/create` action under the provision section in the `teamsapp.yml` file.
+1. Add the following code in the **deploy** section:
+
+   ```
+   deploy:
+     - uses: cli/runNpmCommand # Run npm command
+       with:
+         args: install
+     - uses: cli/runNpmCommand # Run npm command
+       with:
+         args: run build
+     # Deploy bits to Azure Storage Static Website
+     - uses: azureAppService/zipDeploy
+       with:
+         workingDirectory: ./tab
+         # Deploy base folder
+         artifactFolder: .
+         # Ignore file location, leave blank will ignore nothing
+         ignoreFile: .webappignore
+         # The resource id of the cloud resource to be deployed to.
+         # This key will be generated by arm/deploy action automatically.
+         # You can replace it with your existing Azure Resource id
+         # or add it to your environment variable file.
+         resourceId: ${{TAB_AZURE_APP_SERVICE_RESOURCE_ID}}
+     - uses: azureAppService/zipDeploy
+       with:
+         workingDirectory: ./bot
+         # Deploy base folder
+         artifactFolder: .
+         # Ignore file location, leave blank will ignore nothing
+         ignoreFile: .webappignore
+         # The resource id of the cloud resource to be deployed to.
+         # This key will be generated by arm/deploy action automatically.
+         # You can replace it with your existing Azure Resource id
+         # or add it to your environment variable file.
+         resourceId: ${{BOT_AZURE_APP_SERVICE_RESOURCE_ID}}
+   ```
+
+   For more information, see the [sample project](https://github.com/OfficeDev/TeamsFx-Samples/tree/dev/hello-world-bot-with-tab).
+
+1. Run `Teams: Provision` command in Visual Studio Code to apply the bicep to Azure.
+1. Run `Teams: Deploy` command in Visual Studio Code to deploy your app code to Azure.
+1. Open the **Run and Debug Activity Panel** and select **Launch Remote (Edge)** or **Launch Remote (Chrome)**.
+1. Select the F5 key to debug and preview your Teams app.
+
+# [Add message extension to Teams bot app](#tab/botapp)
+
+You can add message extension to a Teams bot app more easily than adding it to a tab app as message extension use the bot support framework.
+
+The following steps help you to add message extension to a bot app:
+
+1. [Create a message extension app using Teams Toolkit](#create-a-message-extension-app-using-teams-toolkit)
+1. [Configure message extension in app manifest](#configure-message-extension-in-app-manifest)
+1. [Bring message extension code to project](#bring-message-extension-code-to-project)
+
+## Create a message extension app using Teams Toolkit
+
+To create a message extension app, see [create a message extension app with Teams Toolkit](create-new-project.md).
```

Comment:
create a message extension app with Teams Toolkit](create-new-project.md). should navigate to app that uses bot functionality.

Reply:
not updated as discussed